pygsp/graphs/nngraphs/twomoons.py

```diff
@@ -44,11 +44,11 @@ class TwoMoons(NNGraph):
 
     """
 
-    def _create_arc_moon(self, N, sigmad, distance, number, seed):
+    def _create_arc_moon(self, N, variance, distance, number, seed):
         rs = np.random.RandomState(seed)
         phi = rs.rand(N, 1) * np.pi
         r = 1
-        rb = sigmad * rs.normal(size=(N, 1))
+        rb = variance * rs.normal(size=(N, 1))
```

Comment:
this should be standard deviation and not variance right?

Reply:
Seems like it. I named it `variance` because the doc said "Variance of the data (do not set it too high or you won't see anything)". Should we name it `stddev` instead? I don't have strong feelings, but a better explanation in the doc will clarify its meaning.